### PR TITLE
Adding a note for extraMounts on Mac and Windows

### PR DESCRIPTION
**Pull request by mauilion**
Signed-off-by: Duffie Cooley <dcooley@apple.com>

### CODE REVIEW

File: site/content/docs/user/configuration.md [hunk 1]
```diff
@@ -226,6 +226,10 @@ for persisting data, mounting through code etc.
 {{< codeFromFile file="static/examples/config-with-mounts.yaml" lang="yaml" >}}
 
 
+NOTE: If you are using Docker for Mac or Windows check that the hostPath is
+included in the Preferences -> Resources -> File Sharing.
+[More Info](https://docs.docker.com/docker-for-mac/#file-sharing)
```
Comment (neolit123) [inline site/content/docs/user/configuration.md:231]: LGTM, one minor nit is that SIG Docs recommend that we do links like this:

> For more information see the [Docker file sharing guide](....).

Comment (mauilion) [reply site/content/docs/user/configuration.md]: updated thanks!

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Comment by BenTheElder** [comment]
/approve
thanks!

**Comment by BenTheElder** [comment]
@aojea fyi podman log export still flakes :(

**Comment by aojea** [comment]
> @aojea fyi podman log export still flakes :(

The error look different this time, error code 255.
I think previously it was an unexpected EOF

**Pull request merged**